book/src/raw-block.md

```diff
@@ -11,22 +11,62 @@ A *mounted (file) volume* is volume that will be mounted using a specified file
 
 The [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md) supports both block and mounted (file) volumes.
 
-While Kubernetes support of mounted (file) volumes is GA/stable, support for [block volume in Kubernetes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#raw-block-volume-support) was introduced in v1.9, and promoted to beta in Kubernetes 1.13.
+While Kubernetes support of mounted (file) volumes is GA/stable, support for [block volume in Kubernetes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#raw-block-volume-support) was introduced in v1.9, and promoted to beta in Kubernetes v1.13. The CSI-specific portions of block volumes was introduced in v1.11 promoted to beta in v1.14.
 
-The Kubernetes CSI tooling support for block volumes is still alpha.
+## Implementing Raw Block Volume Support
 
-## Usage
+CSI doesn't provide a capability query for block volumes, so COs will simply pass through requests for
+block volume creation to CSI plugins, and plugins are allowed to fail with the `InvalidArgument` GRPC
+error code if they don't support block volumes. Kubernetes doesn't make any assumptions about which CSI
+plugins support blocks and which don't, so users have to know if any given storage class is capable of
+creating block volumes.
+
+The difference between a request for a mounted (file) volume and a block volume is the `VolumeCapabilities`
+field of the request. Note that this field is an array and the created volume must support ALL of the
+capabilities requested, or else return an error. If the `AccessType` method of a `VolumeCapability`
+`VolumeCapability_Block`, then the capability is requesting a raw block volume. Unlike mount volumes, block
+volumes don't have any specific capabilities that need to be validated, although access modes still
+apply.
+
+Block volumes are much more likely to support multi-node flavors of `VolumeCapability_AccessMode_Mode`
+than mount volumes, because there's no filesystem state stored on the node side that creates any technical
+impediments to multi-attaching block volumes. There may still be good reasons to prevent multi-attaching
+block volumes, and there may be implementations that are not capable of supporting multi-attach, but it's
+generally wrong to use exactly the same set of access modes for block as for mount volumes.
```

Comment:
instead of generally wrong, I would probably just say something like "evaluate your driver appropriately"